### PR TITLE
PR

### PR DESCRIPTION
**Pull request by anirudhf**
- [x] PR marriage and setup (old images)
- [x] Replace old images with new images
- [x] add hints
- [x] Deprecate mentions of `prefix` use slash commands instead
- [x] new gif for `/imagelock`
- [x] Premium perks mention
- [x] Rename Files
- [x] Remove out-dated hyperlinks and link them to new ones
- [x] Fix some features in `Summary` 
- [x] Add `/hangrygames` in features
- [x] Add `/truthordare` in features
- [x] Use unicode for emojis in badges.md/Make a table

### CODE REVIEW

File: docs/features/profiles/marriages/divorcing.md [hunk 1]
```diff
@@ -2,19 +2,11 @@
...
-
-This method doesn't notify the person of your divorce.
-
-1. Use the `-divorce` command to see a list of the people you're married to.
-2. From the list, pick the number of the person you want to divorce, then do `-divorce number` for example, if I want to divorce the second person on my list, I would use `-divorce 2`.
+1. Use the command `/divorce`.
+2. Click the dropdown menu that appears on the embed, click on the spouse that you want to divorce.
 
 {% hint style="info" %}
-This second method is especially useful if the person you want to divorce no longer shares any mutual servers with you or has deleted their account.
```
Comment (pixxies) [inline docs/features/profiles/marriages/divorcing.md:19]: It's important to mention this is how you divorce someone who isn't in the server. People ask this all the time

File: docs/features/profiles/marriages/marrying.md [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
 
 ## Accepting a marriage proposal
 
-Use `-marry accept @mention` to accept the person's marriage proposal.
+Use `/marry accept <fiancé>` to accept the person's marriage proposal.
 
 ## Canceling a marriage proposal
 
-To cancel a marriage proposal you have made to someone, use `-marry cancel @mention`.
+To cancel a marriage proposal you have made to someone, use `/marry cancel <fiancé>`.
```
Comment (pixxies) [inline docs/features/profiles/marriages/marrying.md]: Here `fiancé` is optional so `[fiancé]`. Running it without the fiancé will give you a dropdown like for divorcing someone off-server.

File: docs/features/profiles/marriages/marrying.md [hunk 3]
```diff
@@ -2,20 +2,20 @@
...
 
 {% hint style="info" %}
-You can see a list of people you're proposed to on your `-profile`.
+You can see a list of people you're proposed to on your `/profile`.
 {% endhint %}
 
 ## Rejecting a marriage proposal
 
-To reject a proposal someone has made to you, do `-marry reject @mention`.
+To reject a proposal someone has made to you, do `/marry decline <user>`.
```
Comment (pixxies) [inline docs/features/profiles/marriages/marrying.md]: User is also optional here and gives a dropdown

**Review by pixxies — CHANGES_REQUESTED** [review]
Looking good! Off to a great start! 🙏

**Comment by pixxies** [comment]
You'll need to add "dropdown" to the `custom-dictionary.txt` file in`.github` and any other words that fail the spellcheck workflow.

**Comment by anirudhf** [comment]
tf so many spellcheck fails 💀 gotta add them

**Comment by anirudhf** [comment]
hello?????? 
image

**Comment by anirudhf** [comment]
YESSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS ALL CHECKS HAVE BEEN PASSED

**Comment by anirudhf** [comment]
You have to edit refunds clause under terms of use, i think that's all

File: docs/features/profiles/marriages/current-marriages.md [hunk 4]
```diff
@@ -3,5 +3,5 @@
 ![](../../../.gitbook/assets/married.jpg)
```
Comment (pixxies) [inline docs/features/profiles/marriages/current-marriages.md]: This image is missing from this PR

**Review by pixxies — DISMISSED** [review]

File: docs/features/profiles/marriages/current-marriages.md (hunk 4 above)
Comment (pixxies) [reply docs/features/profiles/marriages/current-marriages.md]: @Fellintron We just need this and then we're good to go! :)

**Review by pixxies — APPROVED** [review]

**Pull request merged**